### ISSUE TITLE
Installation of latest FUSE fails on High Sierra

### ISSUE THREAD

**Issue by jimrome** [osxfuse/osxfuse#447]
"The installer encountered an error that caused the installation to fail"

**Comment by radj** [comment]
* Is this version 3.7.1?
* Where did you pick it up from?
* Can you spot anything fishy in `/var/log/install.log`?

We've been testing this here on High Sierra and our installs for 3.7.1 DMG downloaded from releases page always succeeds. Can even update fine from System Preferences.

**Comment by jimrome** [comment]
The update to FUSE worked from the preference pane.

**Comment by stvcisco** [comment]
I'm trying a new install and it fails because the signature. Pressing "Allow" on the security preferences does nothing so the install can't complete. (yes, with 3.7.1)

**Comment by stvcisco** [comment]
Interesting... I exited MagicPrefs and tried again, and it allowed the "Allow".  Not sure why that mattered, but it unblocked me.

**Comment by iomz** [comment]
I have the same issue with @stvcisco.
The version is also 3.7.1 but pressing "Allow" button on Security & Privacy > General doesn't do anything.
Not using MagicPrefs.

**Comment by baldr1k** [comment]
I have the same issue as @iomz. Not using MagicPrefs

**Comment by pirana-sas** [comment]
If you use a wacom (as we do) nothing happens when clicking on Allow. But when we clicked it using a normal mouse, it worked. Maybe this will solve it also for you.

**Comment by iomz** [comment]
Although it's not a essential solution, I disabled SPI in Recovery Mode (boot pressing &#8984;+r and `# csrutil disable`) to detour all the third party kernel extension security thing and kept it disabled. So far I'm able to use FUSE like before.

Looks like High Sierra has issues around this KEXT blocking.
Found a thread about VMWare extension: https://discussions.apple.com/thread/8087342

**Comment by cgarrigues** [comment]
I had a similar issue to pirana-sas. I was remotely logged into the system and it didn't work. When I got on the actual screen the click did work.

There is apparently a security check to assure that you're using the local mouse to Allow access.

I guess this makes sense.

**Comment by bfleischer** [comment]
Duplicate of #437.

**Comment by arevaloaraque** [comment]
I used the the keyboard to press "Allow" button and it's works for me. If you press button using click does not works.

**Comment by MichalMisiaszek** [comment]
> I used the the keyboard to press "Allow" button and it's works for me. If you press button using click does not works.

Amazing but true! Thanks

**Comment by ocverclock** [comment]
for me egal

**Comment by stefkes** [comment]
@arevaloaraque  how do you use the keyboard to press Allow? I can't tab on to the button to press enter. Clicking the mouse does nothing.

EDIT: clicking with the mouse works after closing VLC
